.github/workflows/release-please.yml

```diff
@@ -0,0 +1,25 @@
+name: release-please
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: google-github-actions/release-please-action@v3
+        with:
+          release-type: node
+          default-branch: main
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '20.10.0'
+          registry-url: https://registry.npmjs.org/
+      - run: |
+          yarn install --immutable --immutable-cache --check-cache
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
We need to set up this secret then.

Reply:
If the secrets is not set it simply won't publish on NPM. Actually I think it is better removing this, since we won't necessarily create packages.